modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -29,6 +29,18 @@ To view a list of all supported commands, run `rtfctl -h`.
 
 To get more information on a specific command, run `rtfctl <command> -h`.
 
+== Usage
+
+* Install a Mule license
+
+.. Make sure to base64 encode the license following the instructions in xref:install-manual#base64-encode-your-mule-license-key[Base64 Encode your Mule License Key].
```

Comment:
I would add a hyphen between bse64 and encode.

Reply:
Thank you! I know it's weird - it's actually called base64 encode.